packages/worker/.env.example

```diff
@@ -42,6 +42,14 @@ ENABLE_TOKEN_OFFCHAIN_DATA_SAVER=false
 UPDATE_TOKEN_OFFCHAIN_DATA_INTERVAL=86400000
 SELECTED_TOKEN_OFFCHAIN_DATA_PROVIDER=coingecko
 
+BASE_TOKEN_SYMBOL=ETH
+BASE_TOKEN_DECIMALS=18
+BASE_TOKEN_L1_ADDRESS=0x0000000000000000000000000000000000000000
+BASE_TOKEN_ICON_URL=https://assets.coingecko.com/coins/images/279/large/ethereum.png?1698873266
+BASE_TOKEN_NAME=Ether
+BASE_TOKEN_LIQUIDITY=220000000000
```

Comment:
BASE_TOKEN_LIQUIDITY and BASE_TOKEN_USDPRICE are not needed, another service adds this data when token is in the DB.